src/components/blog/Pagination.astro

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
If you look closely at the [...page].astro file, you will notice that the pagination API was implemented but the pagination component was never built. This is because I created this blog when there were only 3 blog posts and couldn't be fussed to create the pagination component at the time.
I blinked and now we have 13 blog posts, so here is the pagination component.
Ta-da!